lua/telescope/builtin/files.lua

```diff
@@ -286,7 +284,7 @@ files.file_browser = function(opts)
           return
         end
 
-        local fpath = current_picker.cwd .. os_sep .. file
+        local fpath = Path:new({current_picker.cwd, file})
```

Comment:
Pretty sure this is wrong. Because we are doing `Path:new(fpath)` later. I haven't tested this but maybe `fpath:` stuff that happens below might just work.

Reply:
Right might be but we also do a string sub in the if. I keep it as string for now :)